docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -275,7 +276,7 @@ function my_plugin_blacklist_blocks() {
 	wp_enqueue_script(
 		'my-plugin-blacklist-blocks',
 		plugins_url( 'my-plugin.js', __FILE__ ),
-		array( 'wp-blocks' )
+		array( 'wp-blocks', 'wp-dom-ready', 'wp-edit-post' )
```

Comment:
Out of curiosity why is `wp-edit-post` required? Looks like wp.domReady and wp.blocks are all that are referenced in the example.

Reply:
It is required because that is where the core block type registrations occur, so the script to unregister needs to be run after. It was explained to some degree about when unregistering block styles, it might be worth repeating again below.

Reply:
Ohhh, got it. I didn't read the whole file but if it's mentioned elsewhere, I'm sure it's fine!